examples/pypi/run.sh

```diff
@@ -0,0 +1,22 @@
+set -x
+set -e
+
+# Start artipie.
+docker run --name artipie -d -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie -p 8080:80 artipie/artipie:latest
+
+# Wait for container to be ready for new connections.
+sleep 5
+
+# Build and upload python project to Artipie
+cd sample-project
+python3 -m pip install --user --upgrade setuptools wheel
+python3 setup.py sdist bdist_wheel
+python3 -m pip install --user --upgrade twine
+python3 -m twine upload --repository-url http://localhost:8080/my-pypi -u username -p 123 dist/*
+cd ..
+
+# Install earlier uploaded python package from artipie.
+python3 -m pip install --index-url http://localhost:8080/my-pypi --no-deps sample_project
```

Comment:
Is it possible to remove `--no-deps` option? I can't find any dependencies declaration in `setup.py`